qmlnotifyservice.h

```diff
@@ -9,18 +9,59 @@ class QQmlApplicationEngine;
 /**
  * Simple notify service for qml.
  * Use :
+ * @code cpp 
```

Comment:
```suggestion
 * @code{cpp} 
```

Reply:
```suggestion
 * @code{cpp} 
```
see the [doxygen](https://www.doxygen.nl/manual/commands.html#cmdcode) documentation